reporter/sbom/spdx.go

```diff
@@ -0,0 +1,587 @@
+package sbom
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand/v2"
+	"slices"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/spdx/tools-golang/spdx"
+	"github.com/spdx/tools-golang/spdx/v2/common"
+
+	"github.com/future-architect/vuls/constant"
+	"github.com/future-architect/vuls/models"
+)
+
+const (
+	creatorOrganization = "future-architect"
+	creatorTool         = "vuls"
+
+	documentSPDXIdentifier = "DOCUMENT"
+	elementOperatingSystem = "Operating-System"
+	elementPackage         = "Package"
+
+	packagePurposeOS          = "OPERATING-SYSTEM"
+	packagePurposeApplication = "APPLICATION"
+	packagePurposeLibrary     = "LIBRARY"
+
+	packageAnnotatorTool = "Tool"
+	annotationOther      = "Other"
+
+	noneField = "NONE"
+
+	relationShipContains = common.TypeRelationshipContains
+	relationShipDepensOn = common.TypeRelationshipDependsOn
```

Comment:
```suggestion
	relationshipContains = common.TypeRelationshipContains
	relationshipDepensOn = common.TypeRelationshipDependsOn
```

"relationship" is a single word.